guide/onboarding/funding-a-wallet.md

```diff
@@ -36,61 +36,19 @@ This section covers a basic mobile Lightning wallet funding flow that does not m
 
-Lightning services greatly simplify the UX of funding and using a Lightning wallet. We have highlighted the sections of funding a wallet with tip boxes like these that can benefit from using Lightning services. We have a dedicated page that covers funding a wallet using the outlined services.
+Lightning services greatly simplify the UX of funding and using a Lightning wallet. We have highlighted the sections of funding a wallet with tip boxes like these that can benefit from using Lightning services.
 
-{% include /tip/close.html %}
-
-<div class="image-slide-gallery">
-
-{% include picture.html
-   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
-   layout = "shadow"
-   caption = "Ask the user if they own bitcoin."
-   alt-text = "Screen asking the user if they own any bitcoin."
-   width = 250
-   height = 541
-%}
-
-{% include picture.html
-   image = "/assets/images/guide/onboarding/funding-a-wallet/funding-request.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/funding-request@2x.png"
-   layout = "shadow"
-   caption = "Show the user a payment request."
-   alt-text = "Screen showing payment request."
-   width = 250
-   height = 541
-%}
+We have a dedicated page that covers funding a wallet using the outlined services.
 
-{% include picture.html
-   image = "/assets/images/guide/onboarding/funding-a-wallet/setting-up.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/setting-up@2x.png"
-   layout = "shadow"
-   caption = "Notify the user once the wallet has received their bitcoin."
-   alt-text = "Screen showing wallet is being setup."
-   width = 250
-   height = 541
-%}
-
-{% include picture.html
-   image = "/assets/images/guide/onboarding/funding-a-wallet/wallet-funded.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/wallet-funded@2x.png"
-   layout = "shadow"
-   caption = "Once funded, the wallet is ready to use."
-   alt-text = "Screen showing that the wallet is now funded."
-   width = 250
-   height = 541
-%}
-
-</div>
+{% include /tip/close.html %}
 
 ## Does the user own bitcoin?
 
-The first step in funding a wallet is asking the user if they currently own bitcoin.
+The first step in funding a wallet is asking the user if they currently own any bitcoin.
 
-If your user does not own bitcoin, your wallet should guide them in obtaining some. One solution is offering a guide the user could read on how to obtain bitcoin. Alternatively, you could offer the option to buy directly within your wallet, which greatly simplifies the funding process.
+If your user does not own bitcoin, your wallet should guide them in obtaining some. Offering a guide on how to obtain bitcoin will prevent users from falling victim to scams and low quality exchanges. Alternatively, you could offer the option to buy directly within your wallet, which greatly simplifies the funding process.
```

Comment:
```suggestion
If your user does not own bitcoin, your wallet should provide practical tips to securely acquire bitcoin from trustworthy exchanges. Alternatively, you could offer the option to buy directly within your wallet, which greatly simplifies the funding process.
```
I think it's better to focus on the desired outcome and avoid the mention of scams.